### PR TITLE
Contacts Documentation

### PR DESCRIPTION
**Pull request by MariusBluem**
@MorrisJobke @nickvergessen @Henni @irgendwie @skjnldsv 

...no worries 😉 Calendar will be next @georgehrke 😜 

Signed-off-by: Marius Blüm <marius@lineone.io>

### CODE REVIEW

File: user_manual/pim/contacts.rst [hunk 1]
```diff
@@ -3,4 +3,107 @@ Using the Contacts App
...
+
+Adding and Managing Address Books
+---------------------------------
+
+Clicking on the settings (gear) button at the bottom of the Apps Information
+field provides access to the Contact app settings. This field shows all
+available address books, certain options for each address book, and enables you
+to create new address books, simply by specifying an address books name.
+
+.. figure:: ../images/contact_manageaddressbook.png
```
Comment (nickvergessen) [inline user_manual/pim/contacts.rst]: image file not readable: pim/../images/contact_manageaddressbook.png

File: user_manual/pim/contacts.rst [hunk 2]
```diff
@@ -3,4 +3,107 @@ Using the Contacts App
...
+After you have set a contact picture, it will look like that:
+
+.. figure:: ../images/contact_picture_set.png
+
+  **Contact picture (set)**
+
+If you want to upload a new one, remove it, view it in full size or download it
+click on the contacts picture.
+
+.. figure:: ../images/contact_picture_options.png
```
Comment (nickvergessen) [inline user_manual/pim/contacts.rst]: We should add that as tooltips to the contacts app, can you create an issue please?

File: user_manual/pim/contacts.rst [hunk 3]
```diff
@@ -3,4 +3,107 @@ Using the Contacts App
...
+
+The Contacts app enables you to create contacts manually.
+
+To create a new contact:
+
+1. Click the ``+ New contact`` button.
+
+   An empty new contact configuration opens in the Application View field.
+
+  .. figure:: ../images/contact_new.png
```
Comment (nickvergessen) [inline user_manual/pim/contacts.rst]: This image looks actually a bit blury?!

File: user_manual/pim/contacts.rst [hunk 4]
```diff
@@ -3,4 +3,107 @@ Using the Contacts App
...
+available address books, certain options for each address book, and enables you
+to create new address books, simply by specifying an address books name.
+
+.. figure:: ../images/contact_manageaddressbook.png
+
+   **Add address book in the contacts settings**
+
+The Contacts settings also enables you to share, export and delete address
+books. You will also find the CardDAV URLs there.
+See :doc:`../pim/` for more details about syncing your adress books
```
Comment (nickvergessen) [inline user_manual/pim/contacts.rst]: This link does not work

File: user_manual/pim/contacts.rst (hunk 2 above)
Comment (MariusBluem) [inline user_manual/pim/contacts.rst]: https://github.com/nextcloud/contacts/issues/183

**Comment by georgehrke** [comment]
> ...no worries 😉 Calendar will be next @georgehrke 😜

\*poke\* 👉
:P

**Comment by MariusBluem** [comment]
@georgehrke all on my to-do list. Will be there with `stable12`-release, as promised 😉🙈

**Comment by MorrisJobke** [comment]
@MariusBluem 12 is coming ... 😉

**Comment by MariusBluem** [comment]
It was a race against time - my MacBook (with a working branch of calendar-docs) is still in repair - so I will backport my changes to stable12 😪 I am so sorry @georgehrke

**Comment by georgehrke** [comment]
> my MacBook (with a working branch of calendar-docs) is still in repair 

No Backups? 😉 🙊

**Comment by jancborchardt** [comment]
cc our RailsGirls Summer of Code students @jonatoni @xh3n1 who will work on the Contacts app & documentation for review. :)

File: user_manual/pim/contacts.rst (hunk 3 above)
Comment (MorrisJobke) [inline user_manual/pim/contacts.rst]: Looks good here.

**Comment by MorrisJobke** [comment]
@MariusBluem Any news?

File: user_manual/pim/contacts.rst (hunk 3 above)
Comment (jonatoni) [inline user_manual/pim/contacts.rst]: Even for me the image looks good.

**Comment by MariusBluem** [comment]
Thanks. Please don’t backport!!! It’s not ready to use 😅

**Pull request merged**